### ISSUE TITLE
Back Button Missing In Search Box

### ISSUE THREAD

**Issue by Kandelai** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3192]
I have upgraded to version 4.6 on Linux (Debian 64bit) and noticed that the new version is missing the back button, which if I remember correctly, appeared in the search bar. Very inconvenient to keep inputting search parameters just to get back to the previous page.
Love the app otherwise!

**Comment by MarshallOfSound** [comment]
Follow the issue template...

If you had you would have checked for duplicate issues and I wouldn't have to deal with this.

**Comment by Kandelai** [comment]
Wow, I always thought developers would welcome bug hunting and outside input. I did search for a similar issue before posting and did not find one, thus following the guidelines. I guess I must have missed the dupe, or it is buried in something. People make mistakes.

You don't have to be an ass about it. How people act, says a lot about their character.

Have a good day.

**Comment by MarshallOfSound** [comment]
@Kandelai I appologise if my response was a bit harsh, however I hav received at least 6 duplicates in the last 2 days for this issue alone and literally a few minutes before you raised this issue I updated the issue template with a checklist for people to follow.  You deleted the checklist and as far as I could tell, ignored it.

Finding the duplicate really isn't that much of a stretch.  See here where I searched for "back button" and all 6 duplicates are the first 6 results...

https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues?q=is%3Aissue+back+button+is%3Aclosed

![screen shot 2018-06-27 at 10 42 05 am](https://user-images.githubusercontent.com/6634592/41946618-c83e330c-79f6-11e8-8d07-220364296c65.png)

> Wow, I always thought developers would welcome bug hunting and outside input

We do, but I welcome people who follow the issue template instead of deleting it significantly more 👍
